microapidev/comment-microapi
# Refactor comments controller methods

**Before submitting your PR for review**

- [x] Run `npm run lint` to find errors in code syntax/format
- [x] Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- [x] Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

The PR refactors the controller methods to be modular and and ensures that a file has a single responsibility. This will improve a 2 key points of interest:

- Code Maintainability (Reduces the time needed to iterate as things are very well defined and the developer can know what needs to be done and how quickly)
- Code Readability (Smaller files with good naming comventions have a smaller cognitive load on the developer and allow code to be easily understood)

**description of Task to be completed?**

- Rename controller methods to be more consistent and well-defined.
- In the controller folder, create a folder named commentsController with an index.js file inside.
- In the commentsController folder, create a js file for each controller method (file is named after the method name).
- Refactor the methods in the original commentsController.js file and place them in their relevant js file created in the previous step.
- Refactor the comments.js file to order the routes in a consistent way.

**How should this be manually tested?**

No business logic was modified during this refactor so test should still pass.

- On your terminal, run `npm run test`.

**Any background context you want to provide?**

Some JSDoc style documentation has been included to improve code documentation.
